statestorage/statestorage_mgr.go

```diff
@@ -55,25 +55,28 @@ func fileExists(filename string) bool {
 	return !info.IsDir()
 }
 
-func InitDb(dbFile string, isNewDB bool) {
-	if (isNewDB == true && fileExists(dbFile)) {
+func InitNewDb(dbFile string) {
+	if (fileExists(dbFile)) {
 		fmt.Printf("\ndataStorageMgr: DB %s already exist, cannot create a new with same name.\n", dbFile)
 		os.Exit(1)
-	} else if (fileExists(dbFile)) {
```

Comment:
(old code)
file should not (does not) exist -- so we do not take this branch either.
Maybe the logic is correct if else if  NOT fileExists, but I think the refactoring is better.